structure/structure_test.go

```diff
@@ -352,6 +337,6 @@ func (*testTxStructureSuite) TestError(c *C) {
 	}
 	for _, err := range kvErrs {
 		code := terror.ToSQLError(err).Code
-		c.Assert(code != mysql.ErrUnknown && code == uint16(err.Code()), IsTrue, Commentf("err: %v", err))
+		require.True(t, code != mysql.ErrUnknown && code == uint16(err.Code()), "err: %v", err)
```

Comment:
It can be replaced with two concrete `require`: NotEqual and Equal.

Reply:
ok, replaced.